torch/csrc/jit/codegen/cuda/ir_interface_nodes.h

```diff
@@ -548,6 +589,11 @@ class TORCH_CUDA_CU_API TensorView : public Val {
   //!  This is the temporary flag for indicating that the new swizzle
   //!  implementation is used and will be removed in follow ups.
   bool has_swizzle_op_ = false;
+
+  // Indicates if the lowering pass should try to pre-allocate
+  //   and pre-compute address for this tensor.
+  bool lift_read_address_ = false;
+  bool lift_write_address_ = false;
```

Comment:
Haven't read the whole PR yet, but why do we want to have two flags? What's the difference of read and write addresses?